spec.html

```diff
@@ -4069,7 +4069,7 @@ <h1>Await</h1>
       <emu-clause id="sec-normalcompletion" type="abstract operation">
         <h1>
           NormalCompletion (
-            _value_: unknown,
+            _value_: any value except a Completion Record,
```

Comment:
can this take spec enums and Lists and stuff, or only ECMAScript language values?

Reply:
PR #2547 changed the declared type of the `[[Value]]` field of Completion Records from "an ECMAScript language value or `~empty~`" to "any value except a Completion Record", so it's certainly well-behaved for `NormalCompletion` to take non-language values.

Mind you, the only obvious example of this happening (that I could find) is 3 occurrences of `NormalCompletion(~unused~)`. It probably happens mostly via Implicit Normal Completion (e.g., in the return from CreateListFromArrayLike).